### PR TITLE
Births to WLHIV

### PR DESCRIPTION
**Pull request by mwalte10**
Calculates the number of births to WLHIV, needed for MTCT. Please review the 15 year old entrants bit that has changed in the demographic model in particular to make sure that won't cause any issues. 

Included births in test child file to make sure it is never negative.

### CODE REVIEW

File: inst/include/hiv_demographic_projection.hpp [hunk 1]
```diff
@@ -42,17 +42,37 @@ void run_age_15_entrants(int time_step,
...
+  }
+  for (int g = 0; g < ss.NS; ++g) {
+    for (int hm = 0; hm < hc_ss.hc2DS; ++hm) {
+      for (int hu = 0; hu < ss.hTS; ++hu) {
+        intermediate.children.age15_art_pop(hu, hm, g) += state_curr.children.hc2_art_pop(hu, hm, (hc_ss.hc2AG-1), g);
       }
     }
   }
+
+  if(time_step == 41 ){
```
[the remark this answers is not in the document]
Comment (r-ash) [reply inst/include/hiv_demographic_projection.hpp:62]: Oh is this a bit of old debug code?

File: inst/include/hiv_demographic_projection.hpp [hunk 2]
```diff
@@ -99,24 +121,40 @@ void run_hiv_and_art_stratified_ageing(int time_step,
   }
 
   // TODO: add HIV+ 15 year old entrants see https://github.com/mrc-ide/leapfrog/issues/8
```
Comment (r-ash) [inline inst/include/hiv_demographic_projection.hpp:123]: Can this TODO be removed now?

**Review by r-ash — APPROVED** [review]
This looks great, couple of small suggestions but they are fairly inconsequential. I'm approving so not blocked on me whilst I am away next week but would be worth a review from science side before merging this.

File: inst/include/hiv_demographic_projection.hpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (mwalte10) [reply inst/include/hiv_demographic_projection.hpp:62]: good catch! removed

**Comment by mwalte10** [comment]
@jeffeaton  I've added you to look through the methods per Rob's comment of approval from the technical side. I'm incorporating his suggestions but let me know if you have any others.

**Pull request closed without merging**